### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Arcoders**

### CODE REVIEW

File: staff/ismael-haytam/app-template/home.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <li><a href="./about.html">About</a></li>
+            <li><a href="./profile.html">Profile</a></li>
+            <li><a href="./contact.html">Contact</a></li>
+        </ul>
+    </nav>
+    <header>
+        <h1>Quas doloribus sit rerum minus</h1>
+        <img src="https://picsum.photos/200/300" alt="random image">
+    </header>
+    <section>
```
Comment (vior1992) [inline staff/ismael-haytam/app-template/home.html]: You can use "main" for englobe inside all "sections", nice code by the way.

File: staff/ismael-haytam/app-template/news.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>News</title>
+</head>
+<body>
+
```
Comment (HenryNovoa) [inline staff/ismael-haytam/app-template/news.html:10]: A nav bar would be nice

File: staff/ismael-haytam/app-template/home.html (hunk 1 above)
Comment (Arcoders) [reply staff/ismael-haytam/app-template/home.html]: thanks ^^

File: staff/ismael-haytam/app-template/news.html (hunk 2 above)
Comment (Arcoders) [reply staff/ismael-haytam/app-template/news.html:10]: Okeeeey ^^

File: staff/ismael-haytam/app-template/news.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        </article>
+        <article>
+            <h2>Lorem ipsum dolor set amet</h2>
+            <img src="https://picsum.photos/200/300" alt="random image">
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Provident aut asperiores consectetur corrupti. Corrupti dolorum iste molestias in, nostrum officia, dolores quas mollitia eaque eligendi accusantium deserunt exercitationem, quo a.</p>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat cum harum dolorum porro fuga vel, consequatur explicabo nihil veritatis unde repudiandae officiis rerum maiores rem dolore voluptate obcaecati. Praesentium, eveniet!</p>
+            <ul>
+                <li>Compartir: </li>
+                <li><a href="http://www.facebook.com">Facbook</a></li>
+                <li><a href="http://www.twitter.com"">Twiter</a></li>
```
Comment (diesttro) [inline staff/ismael-haytam/app-template/news.html]: There is an extra quotation mark

**Pull request merged**